tests/rptest/redpanda_cloud_tests/cloudv2_object_store_blocked.py

```diff
@@ -1,12 +1,22 @@
+from rptest.services.redpanda_cloud import CLOUD_TYPE_BYOC, CLOUD_TYPE_FMC
+
+
 class cloudv2_object_store_blocked:
     """Temporary block writes to a cloudv2 TS bucket"""
     def __init__(self, rp, logger):
         self.logger = logger
         cluster_id = rp._cloud_cluster.config.id
         network_id = rp._cloud_cluster.current.network_id
         self._cloud_provider_client = rp._cloud_cluster.provider_cli
-        self._vpc_id = self._cloud_provider_client.get_vpc_by_network_id(
-            network_id)['VpcId']
+        # This is different for BYOC and FMC
+        if rp._cloud_cluster.config.type == CLOUD_TYPE_BYOC:
+            self._vpc_id = self._cloud_provider_client.get_vpc_by_network_id(
+                network_id)['VpcId']
+        elif rp._cloud_cluster.config.type == CLOUD_TYPE_FMC:
```

Comment:
did we decide to add preprod creds for FMC testing? otherwise we will want to skip this test if we're testing an FMC cluster

Reply:
Yes, we should skip this in case of FMC. That will be the part of the different PR